### ISSUE TITLE
Visualising "simplified" tree sequences

### ISSUE THREAD

**Issue by hyanwong** [kitchensjn/tskit_arg_visualizer#16]
If we simplify a tree sequence so as to collapse the recombination nodes down into the nearest recombination node, will the visualisation work? This might be needed to make plots of genealogies such as those that are inferred by Relate or tsinfer e.g. https://github.com/tskit-dev/what-is-an-arg-paper/issues/18#issuecomment-1079978538

**Comment by kitchensjn** [comment]
You can view simplified tree sequences if you set 'line_type="line"'. Nodes with two parents are not the true recombination nodes in this scenario. 'line_type="ortho"' should be reserved for full ARGs as the visualizer does not provide the correct edge pathing with simplified sequences, particularly for nodes with two parents and two children. It's probably worth adding a mark that says whether you are visualizing a tree sequence or full ARG, as at first glance, those two objects can look pretty similar in the visualizer.

**Comment by hyanwong** [comment]
Great, thanks for the clarification. It would be worth documenting this, e.g. in the `line_type` parameter, I guess?

Lots of simulated tree sequences "in the wild" are likely to be of the simplified type (e.g. many with > 2 parents), so it's probably worth giving advice for that use-case.

Thanks again for your work on this. It's fabulous.

**Comment by kitchensjn** [comment]
I've changed the `line_type` parameter to `edge_type` to better fit the terminology that I've been using in the documentation and added a note about scenarios to use "ortho" versus "line".

If it would be helpful to implement the "ortho" edge type for "simplified" tree sequences (or any sequence that doesn't conform to the pattern return by `msprime.sim_ancestry(..., record_full_arg=True)`), we would need to consider two thing:

1) The visualizer uses the msprime node flags to determine the node type and thus number of parents/children. It would be more robust to check this count, in the event that the nodes are not flagged in the same manner.
2) It assumes that each node has at most two parents or two children. The `alt_parent`/`alt_child` attributes for each edge in the JSON take a single ID (see (plotting.md)[https://github.com/kitchensjn/tskit_arg_visualizer/blob/main/plotting.md]). This could be expanded to be a list of alternative parents/children instead. I had started to work on this with #9, but I haven't yet worked through the new pathing types that are needed, particularly in scenarios where a node has both multiple parents and multiple children.
